lambda/getBinInterval.ts

```diff
@@ -15,14 +16,13 @@ export const getBinInterval =
 		durationHours: number
 		iccid: string
 	}): Promise<Record<string, unknown>[]> => {
```

Comment:
**problem:** Do not use `unknown` here. You know exactly the type of the returned result.